qa/standalone/ceph-helpers.sh

```diff
@@ -476,10 +476,15 @@ function test_run_mon() {
 
 function create_rbd_pool() {
     ceph osd pool delete rbd rbd --yes-i-really-really-mean-it || return 1
-    ceph osd pool create rbd $PG_NUM || return 1
+    create_pool rbd $PG_NUM || return 1
     rbd pool init rbd
 }
 
+function create_pool() {
+    ceph osd pool create "$@"
+    sleep 1
```

Comment:
@liewegas It seems to have been working reliably with the python teuthology tests.  I think it would be better to add a wait for pool.  It would wait on map changes and check for the new pool name until it appears.